### PR TITLE
add fils for nodejs week3 homework

### PR DESCRIPTION
**Pull request by aladdin91**
this is my nodejs week 3 warmup h/w

### CODE REVIEW

File: nodejs/week3/app.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const knex = require("knex")({
+  client: "mysql2",
+  connection: {
+    host: process.env.DB_HOST || "127.0.0.1",
+    port: process.env.DB_PORT || 3306,
+    user: process.env.DB_USER || "root",
+    password: process.env.DB_PASSWORD || "123456789",
```
Comment (Divyajg) [inline nodejs/week3/app.js:7]: Just a gentle reminder to use a .env file for sensitive data like passwords. It is okay in this case.

**Review by Divyajg — COMMENTED** [review]
Good work. 
But you haven't attached the recording of SQL injection demo!

**Comment by aladdin91** [comment]
i add it in the secend commit
https://github.com/aladdin91/hyf-homework/pull/20/commits/0e9a24d5da2cdfce4c006426e5bb50c99c409f97

**Comment by aladdin91** [comment]
thanks a lot for the review 🙏

**Pull request merged**